maquetado/scss/detail.scss

```diff
@@ -0,0 +1,22 @@
+.title {
+  margin: 20px 0 0 0;
+}
+
+.field {
+  font-size: 19px;
+  font-weight: bold;
+}
+
+.field-description {
+  color: gray;
```

Comment:
create a scss file called colors.scss and specify all the colors (with the hex value) you are going to use there